strings/ion_laws.json

```diff
@@ -445,7 +445,7 @@
         "SECURITY OFFICERS",
         "WARDENS",
         "DETECTIVES",
-        "LAWYERS",
+		"CORPORATE AFFAIRS AGENTS",
```

Comment:
Is this a github moment or is this a random indent here?

Reply:
github moment